buildpack.md

```diff
@@ -432,13 +434,13 @@ Correspondingly, each `/bin/build` executable:
 - MAY read or write to the `<app>` directory.
 - MAY read the build environment as described in the [Environment](#environment) section.
 - MAY read the Buildpack Plan.
-- MAY augment the Buildpack Plan with more refined metadata.
-- MAY remove entries with duplicate names in the Buildpack Plan to refine the metadata.
-- MAY remove all entries of the same name from the Buildpack Plan to defer those entries to subsequent `/bin/build` executables.
+- SHOULD write a list containing any unmet Buildpack Plan entries to `<layers>/build.toml` to defer those entries to subsequent `/bin/build` executables.
```

Comment:
I'm not sure if we define `unmet` anywhere in the spec? We explain what happens to unmet entries below, but might be good to define this term before it's used.

Reply:
Let me know if the changes in [this commit](https://github.com/buildpacks/spec/pull/139/commits/61bf166cac1a08c73455b878f97a14a3d929d53e) address your concern.

Related: I think the Buildpack API would generally benefit from a terminology section but that change is probably out of scope for this PR. I created an issue for this change - https://github.com/buildpacks/spec/issues/145